docs/en/basic/Command-Line-Interface.md

```diff
@@ -132,105 +131,104 @@ $ ./bin/alluxio getConf --unit S alluxio.master.journal.flush.timeout
 
 ### logLevel
 
-The `logLevel` command allows you to get or change the log level of a particular class on specific
-instances. Our users are able to change Alluxio server-side log level at runtime using this command.
+The `logLevel` command returns the current value of or updates the log level of a particular class
+on specific instances. Users are able to change Alluxio server-side log levels at runtime.
 
-The syntax is `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`.
-* `--logName <arg>` indicates the logger's name (e.g.alluxio.master.file.DefaultFileSystemMaster)
-* `--target <arg>` lists the Alluxio master or workers to set. The target could be
-`<master|workers|host:webPort>`. A list of targets separated by `,` can be specified. `host:webPort`
-pair must be one of the workers. Default target is master and all workers.
-* `--level <arg>` If provided the command changes to the defined logger level, 
-otherwise it gets and displays the current logger level.
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+where:
+* `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
+* `--target <arg>` lists the Alluxio master or workers to set.
+The target could be of the form `<master|workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
+`host:webPort` pair must be one of the workers. TODO: what does this mean??? must be???
```

Comment:
-`host:webPort` pair must be one of the workers. TODO: what does this mean??? must be???
+The `host:webPort` form only works with worker hostnames and ports.